README.md

```diff
@@ -142,4 +142,4 @@ Because we don't upgrade vendor pods every day, even once in a few months, the c
 # License
 
 The cocoapods-binary-cache plugin is available as open-source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
-It uses [cocoapods-rome](https://github.com/CocoaPods/Rome) and [cocoapods-binary](https://github.com/leavez/cocoapods-binary) internally, which are also under MIT License.
\ No newline at end of file
+It uses [cocoapods-rome](https://github.com/CocoaPods/Rome) and [cocoapods-binary](https://github.com/leavez/cocoapods-binary) internally, which are also under MIT License.
```

Comment:
I edited the file through browser on Github and I think there wasn't a new-line character at the end of the file and Github might have added that